docs/modeling-your-data/modeling-your-data-with-dbt/dbt-quickstart/normalize/index.md

```diff
@@ -114,8 +114,37 @@ Depending on the use case it should either be the catalog (for Unity Catalog use
 
 :::
 
+### 10. Partition Timestamp Configuration
 
-### 10. Run your model(s)
+The package uses a configurable partition timestamp column, controlled by the `snowplow__partition_tstamp` variable:
+
+```yaml
+vars:
+  snowplow__partition_tstamp: "collector_tstamp"  # Default value
+```
```

Comment:
Not sure if we wanted to highlight that it might make sense to change this to derived_tstamp if they wanted to make that as the main key to incrementalize on and make it optimal to filter on (as it is the closest to the actual time when the event happened)

Reply:
It basically needs an explanatory sentence which says what this is for and where it gets applied. It should say that it is for the normalized tables only and that it is the main field it will get incrementalized on. You can say it may make sense to change this to the derived_tstamp if that's what you want to report on downstream or sth.

Reply:
OK I think we phrase it in such a way